restful_api_with_data_mapper/main.bal

```diff
@@ -0,0 +1,95 @@
+import ballerina/http;
+
+type Person record {
+    readonly string id;
+    string firstName;
+    string lastName;
+    int age;
+    string country;
+};
+
+type Course record {
+    string id;
+    string name;
+    int credits;
+};
+
+type Student record {
+    string id;
+    string fullName;
+    string age;
```

Comment:
Is it meaningful to use age as string?

Reply:
Fixed via [74db4ed](https://github.com/ballerina-guides/integration-samples/pull/27/commits/74db4ed66a889c18321790d55a4092e0647cf9b3)